frontend/src/context/datasetStateSlice.ts

```diff
@@ -247,7 +247,7 @@ export const loadAdminBoundaryDataset = async (
 
   const hdcRequestParams = {
     level,
-    admin_id: adminCode,
+    id_code: adminCode,
```

Comment:
Is this change necessary? Do we know what the most common standard is?
admin_id vs id_code?

Linked to my comment on URLs, ideally the frontend stays indifferent as to where the data is coming from.